### ISSUE TITLE
Getting started tutorial broken by recent piston graphics library update(s)

### ISSUE THREAD

**Issue by ChrisPWill** [PistonDevelopers/Piston-Tutorials#55]
Hi, the recent updates to the graphics libraries break the tutorial, giving this:

```
src/main.rs:45:46: 45:60 error: mismatched types:
 expected `[[f64; 3]; 2]`,
    found `&graphics::context::Context`
(expected array of 2 elements,
    found &-ptr) [E0308]
src/main.rs:45             graphics::rectangle(RED, square, center_context, gl);
                                                            ^~~~~~~~~~~~~~
```

I'd fork the repo myself and fix it, but I'm unable to figure out where in the graphics library graphics::rectangle is (and thus I can't figure out what new arguments are required here). It's probably something simple, but I'm too new to Rust to spot it.

**Comment by bvssvni** [comment]
It is here https://github.com/PistonDevelopers/graphics/blob/master/src/lib.rs#L118

`center_context.transform` should fix it.

**Comment by ChrisPWill** [comment]
Yep, you're right, it really was as simple as that. The tutorial example is working again with that fix. Thanks.

**Comment by Potpourri** [comment]
Fixed by #56
